### PR TITLE
Add email to FutureWallet

### PR DESCRIPTION
**Pull request by JustynaBroniszewska**

### CODE REVIEW

File: universal-login-commons/src/core/models/wallets/SerializableFutureWallet.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ export interface SerializableFutureWallet {
   gasPrice: string;
   ensName: string;
   gasToken: string;
+  email: string;
```
Comment (xitronix) [inline universal-login-commons/src/core/models/wallets/SerializableFutureWallet.ts:10]: should be opitonal

**Review by xitronix — DISMISSED** [review]

File: universal-login-commons/src/core/models/wallets/SerializableFutureWallet.ts [hunk 2]
```diff
@@ -7,6 +7,7 @@ export interface SerializableFutureWallet {
   gasPrice: string;
   ensName: string;
   gasToken: string;
+  email?: string;
```
Comment (JustynaBroniszewska) [inline universal-login-commons/src/core/models/wallets/SerializableFutureWallet.ts:10]:
```suggestion
  email?: string
```

File: universal-login-sdk/test/core/services/WalletService.unit.test.ts [hunk 3]
```diff
@@ -70,6 +70,7 @@ describe('UNIT: WalletService', () => {
       ensName: 'justyna.mylogin.eth',
       gasPrice: TEST_GAS_PRICE,
       gasToken: ETHER_NATIVE_TOKEN.address,
+      email: '',
```
Comment (xitronix) [inline universal-login-sdk/test/core/services/WalletService.unit.test.ts:73]: use undefined or email string

Comment (JustynaBroniszewska) [reply universal-login-sdk/test/core/services/WalletService.unit.test.ts]: 👍

**Review by xitronix — APPROVED** [review]

**Pull request merged**